documentation/assemblies/deploying/assembly-deploy-intro-operators.adoc

```diff
@@ -7,27 +7,25 @@
 = Strimzi operators
 
 [role="_abstract"]
-Strimzi operators are purpose-built with specialist operational knowledge to effectively manage Kafka on Kubernetes.
-Each operator performs a distinct function.
-
-Cluster Operator:: The Cluster Operator handles the deployment and management of Apache Kafka clusters on Kubernetes. 
-It automates the setup of Kafka brokers, and other Kafka components and resources.
-Topic Operator:: The Topic Operator manages the creation, configuration, and deletion of topics within Kafka clusters. 
-User Operator:: The User Operator manages Kafka users that require access to Kafka brokers. 
-
-When you deploy Strimzi, you first deploy the Cluster Operator.
-The Cluster Operator is then ready to handle the deployment of Kafka.
-You can also deploy the Topic Operator and User Operator using the Cluster Operator (recommended) or as standalone operators.
-You would use a standalone operator with a Kafka cluster that is not managed by the Cluster Operator.
-
-The Topic Operator and User Operator are part of the Entity Operator.
-The Cluster Operator can deploy one or both operators based on the Entity Operator configuration.
-
-[IMPORTANT]
-====
-To deploy the standalone operators, you need to set environment variables to connect to a Kafka cluster.
-These environment variables do not need to be set if you are deploying the operators using the Cluster Operator as they will be set by the Cluster Operator.
-====
+Strimzi uses operators to deploy and manage Kafka components.
+The operators monitor Strimzi custom resources (like `Kafka`, `KafkaTopic`, and `KafkaUser`) and ensure that the Kafka components are configured and running as specified in the resources.
+
+Strimzi provides the following operators, each handling different aspects of a Kafka deployment:
+
+Cluster Operator (required):: Handles the deployment and management of Apache Kafka clusters on Kubernetes. It automates the setup of Kafka nodes, and other Kafka components and resources.
+Entity Operator (recommended):: Incorporates the following to manage topics and users:
+* *Topic Operator* for managing Kafka topics.
+* *User Operator* for managing Kafka users.
+ifdef::Section[]
+Access Operator (optional):: Helps manage and share Kafka cluster connection details.
+endif::Section[]
+
+Additionally, Strimzi provides Drain Cleaner, which supports the Cluster Operator in managing pod evictions for Kafka clusters.
+
+The Cluster Operator is required and must be deployed first.
+It handles the deployment of Kafka and, if configured, can also deploy the Entity Operator, which may include the Topic Operator, User Operator, or both.
+
+The Topic Operator and User Operator can also be deployed standalone to manage topics and users for a Kafka cluster that is not managed by Strimzi.
```

Comment:
If we want to list the Access Operator and Drain Cleaner in the introduction in sich a prominent way, we should probably also cover here how they are installed.

Reply:
+1